### ISSUE TITLE
<div> before html elements (select, text, checkbox, etc..)

### ISSUE THREAD

**Issue by LukaszFokin** [dwightwatson/bootstrap-form#50]
Im trying to use inline form using bootstrap-form. But at all html elements create using BootForm, all elements are displayed between div tag. So, I can't get label and text at the same line.

I made some tests and if removed div tag from BootstrapForm.

**Comment by dwightwatson** [comment]
Yeah, this package is better suited for horizontal forms, the vertical form support needs some work. The code would need to be adjusted to only create the surrounding `<div>` for horizontal form groups. Happy to take a look at any PRs that can sort this out.

**Comment by LukaszFokin** [comment]
Tks for reply, I'll take a look.
